src/_P068_SHT3x.ino

```diff
@@ -90,6 +97,12 @@ void SHT3X::readFromSensor()
     {
       tmp = ((((data[0] << 8) | data[1]) * 175.0f) / 65535.0f) - 45.0f;
       hum = ((((data[3] << 8) | data[4]) * 100.0f) / 65535.0f);
+      // Humidity temperature compensation borrowed from P028 BME280
+      if ((tmpOff > 0.1f) || (tmpOff < -0.1f)) {
```

Comment:
We now also have a proper compare function for floating point values, borrowed from Donald Knuth's books :)
See `Helpers/ESPEasy_math.h`